source/vm.cc

```diff
@@ -93,6 +93,10 @@ namespace phoenix {
 		pop_call();
 	}
 
+	void vm::unsafe_jump(uint32_t address) {
+		this->jump(address);
+	}
```

Comment:
What purpose does `unsafe_jump` have?

Reply:
`repeat` and `while` loop in ikarus do require jump from naked call. Decide to provide clear statement that `jump` is not normal use of VM.

Alternative is to make `jump` a public function

Reply:
Okay, that's fine then :)